src/shims/foreign_items.rs

```diff
@@ -926,7 +926,7 @@ pub trait EvalContextExt<'mir, 'tcx: 'mir>: crate::MiriEvalContextExt<'mir, 'tcx
                 // args[0] : LPCWSTR lpName (32-bit ptr to a const string of 16-bit Unicode chars)
                 // args[1] : LPCWSTR lpValue (32-bit ptr to a const string of 16-bit Unicode chars)
                 // Return nonzero if success, else return 0.
-                
+                throw_unsup_format!("can't call foreign function: {}", link_name);
```

Comment:
Given that this is a specialized branch you could also give a more targeted error message, such as `can't set environment variable on Windows` or so.

Reply:
I can fix it right away! Lousy me 😂